backend-go/config.yaml

```
@@ -0,0 +1,46 @@
+Server:
+  Port: 3001
+
+Auth:
+  Github:
+    # WARNING: Replace these values with your GitHub OAuth credentials
+    ClientID: ""
+    ClientSecret: ""
+    CallbackURL: "http://localhost:3000/auth/login/github"
+    AuthorizationURL: "https://github.com/login/oauth/authorize"
+    TokenURL: "https://github.com/login/oauth/access_token"
+    UserProfileURL: "https://api.github.com/user"
+  FrontendBaseURL: "http://localhost:5173"
+
+JWT:
+  # WARNING: Use strong, unique secrets in production
+  AccessTokenSecret: "<STRONG_RANDOM_SECRET>"
+  AccessTokenExpirationTime: "24h"
+  RefreshTokenSecret: "<STRONG_RANDOM_SECRET>"
+  RefreshTokenExpirationTime: "168h"
+
+Yorkie:
+  APIAddr: "http://localhost:8080"
+  ProjectName: "default"
+  ProjectSecretKey: ""
+
+Mongo:
+  ConnectionTimeout: "10s"
+  ConnectionURI: "mongodb://localhost:27017/codepair"
+  PingTimeout: "5s"
+  DatabaseName: "codepair"
+
+Storage:
+  Provider: "minio"       # or "s3"
```

how about listing all available options for this? like:
```suggestion
  Provider: "minio"       # "minio" | "s3"
```

---

Thank you for good suggestion. I've applied it!